### PR TITLE
Added social media icons in header

### LINKED ISSUE #961: Add Social Media icons in menu on top right
Please add the following social media icons in the top right:
* Twitter
* Facebook
* Github
* YouTube

### PR DESCRIPTION
**Pull request by ankurg22**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a Preview link and screenshot showing after and before the changes.
- [ ] Images are `240 x 240` [w x h].
- [x] Included a description of change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- Social medias like Facebook, Twitter, Github, Youtube are added in the
top right corner of the header.
- https://ankurg22.github.io/gci17.fossasia.org/
- Closes #961

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -88,7 +88,27 @@
 										<a href="https://gci14.fossasia.org">FOSSASIA GCI 2014/15</a>
 									</li>
 								</ul>
-							</li>	
+							</li>
+              <li>
```
Comment (abishekvashok) [inline _includes/header.html]: Can you please intend the code? It will be great. Rest looks fine.

**Review by abishekvashok — CHANGES_REQUESTED** [review]

File: _includes/header.html (hunk 1 above)
Comment (ankurg22) [reply _includes/header.html]: Sure. Will make changes :blush:

**Review by rupav — COMMENTED** [review]
Please indent the code, rest LGTM

**Comment by ankurg22** [comment]
@abishekvashok @rupav Done. Please review again :blush:

**Review by sayak119 — APPROVED** [review]

File: _includes/header.html [hunk 2]
```diff
@@ -17,7 +17,7 @@
 	<link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" type="text/css" media="all">
 	<link href='https://fonts.googleapis.com/css?family=Lato:300,400%7CRaleway:100,400,300,500,600,700%7COpen+Sans:400,500,600' rel='stylesheet' type='text/css'>
 	<link rel="icon" href="favicon.ico" />
-
+
```
Comment (ParthS007) [inline _includes/header.html]: Remove whitespace

File: _includes/header.html [hunk 3]
```diff
@@ -39,7 +39,7 @@
 							<li>
 								<a href="{{ baseurl }}./#welcome">Welcome</a>
 							</li>
-     						<li>
+							<li>
```
Comment (ParthS007) [inline _includes/header.html:41]: Remove whitespace

**Review by ParthS007 — CHANGES_REQUESTED** [review]

File: _includes/header.html (hunk 2 above)
Comment (ankurg22) [reply _includes/header.html]: Sure.

File: _includes/header.html (hunk 3 above)
Comment (ankurg22) [reply _includes/header.html:41]: Why this one? The code is properly indented here.

Comment (rupav) [reply _includes/header.html:41]: This is fine.

**Comment by ankurg22** [comment]
@parths007 Please review now.

**Comment by rupav** [comment]
Good work @ankurg22

**Comment by gabru-md** [comment]
@rupav please usee Squash and Commit.
This avoids an additional merge commit in the logs.
:+1:
I've already told this to you.
But please make sure about this

**Comment by rupav** [comment]
got it .

**Pull request merged**